example/application/fixtures/v2/opencerts.tsx

```diff
@@ -115,7 +115,7 @@ export const rawOpencerts: OpencertsDocuments = {
 };
 
 export const wrappedOpencerts: WrappedDocument<OpencertsDocuments> = {
-  data: document,
+  data: document as any,
```

Comment:
if document as type OpenCertsDocuments it should work. what's the issue ?

Reply:
after upgrading open-attestation, there is a type clash:
`Type 'Document' is missing the following properties from type 'DeepStringify<OpencertsDocuments>': additionalData, description, issuers`

Reply:
hum ... ok I'm lazy to investigate because it's just an example but it shouldn't be done that way ..